s4/clarity/lims.py

```diff
@@ -91,61 +92,102 @@ def __init__(self, root_uri, username, password, dry_run=False, insecure=False,
         from .configuration.stage import Stage
         from .lab import Lab
 
+        # Initialise the list of factories. The element factories will
+        # add themselves to this dictionary when they are created
         self.factories = {}
 
-        # there's no need to make these lazy, we are probably using at least a couple of them
-        self.steps = StepFactory(self, Step, batch_flags=BatchFlags.QUERY)
+        self.steps = StepFactory(self, Step)
```

Comment:
This seems like an awful lot of blank space. Now that they are so short I would remove all the blank space.